tests/GameTests.cpp

```diff
@@ -11,3 +13,19 @@ TEST_F(GameTests, AddFrames)
     game.addFrames("Someone:32|4");
 }
 
+TEST_F(GameTests, conversion_to_int)
+{
+    Game game;
```

Comment:
Jeśli do struktury wrzuciłeś zmienną `game` to tutaj nie jest ona potrzebna. Im mniej kodu tym lepiej. Tę linię można usunąć.